workflow-core/src/commonMain/kotlin/com/squareup/workflow1/StatefulWorkflow.kt

```diff
@@ -93,6 +92,18 @@ public abstract class StatefulWorkflow<
     snapshot: Snapshot?
   ): StateT
 
+  /**
+   * @see [SessionWorkflow.initialState].
```

Comment:
Do we really need to add this? Can the caller instead do an `is SessionWorkflow` check? If so, remember to delete the last paragraph of the `SessionWorkflow` class kdoc.

Reply:
We call this from the runtime in `WorkflowNode` - which is all based on `StatefulWorkflow`, so yes, I think we need this unless we want to switch the order of the hierarchy.